### PR TITLE
Regenerate client using Stone 3.2.0

### PR DESCRIPTION
**Pull request by yuxiang-he**
## **Checklist**

**General Contributing**
- [ ] Have you read the Code of Conduct and signed the [CLA](https://opensource.dropbox.com/cla/)?

**Is This a Code Change?**
- [ ] Non-code related change (markdown/git settings etc)
- [x] SDK Code Change
- [x] Example/Test Code Change

**Validation**
- [x] Does `tox` pass?
- [x] Do the tests pass?

## Changes
* Specs are regenerated using Stone 3.2.0
* Fix relative import issues with the breaking change introduced in Stone 2.0.0
* [breaking change] `dropbox.dropbox` is now renamed to `dropbox.dropbox_client`
* `test/requirements.txt` now needs Stone as well

### CODE REVIEW

**Comment by yuxiang-he** [comment]
I think we can just skip generating `stone_base.py` `stone_validators.py` and `stone_serializers.py`, but that would most likely be another breaking change in tone

**Review by rogebrd — COMMENTED** [review]
Since this is breaking, can you create an `UPGRADING.md` file similar to [UPGRADING.md](https://github.com/dropbox/dropbox-sdk-js/blob/main/UPGRADING.md) in order to make sure people know how to upgrade to the new SDK?

Side Note: Glad to see all the removed code

File: UPGRADING.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Upgrading the Dropbox SDK
+
+This document is designed to show you how to upgrade to the latest version of the SDK accomodating any breaking changes introduced by major version updates.  If you find any issues with either this guide on upgrading or the changes introduced in the new version, please see [CONTRIBUTING.md][contributing]
```
Comment (rogebrd) [inline UPGRADING.md:3]: This will need a contributing link similar to the code of conduct at the bottom. Otherwise the contributing link won't go anywhere

**Review by rogebrd — APPROVED** [review]
Looks good overall, just need to update the link in UPGRADING.md

**Pull request merged**